boards/nano_rp2040/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "nano_rp2040"
```

Comment:
Seems like we should include the "connect" in case they create a non wireless version.

Reply:
I renamed the crate